dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -277,21 +272,11 @@ private Set<TrackedEntityAttributeValue> getTrackedEntityAttributeValues(Enrollm
   }
 
   private List<Enrollment> findEnrollments(
-      Iterable<Enrollment> enrollments,
-      EnrollmentFields fields,
-      boolean includeDeleted,
-      OrganisationUnitSelectionMode orgUnitMode) {
+      Iterable<Enrollment> enrollments, EnrollmentFields fields, boolean includeDeleted) {
     List<Enrollment> enrollmentList = new ArrayList<>();
-    UserDetails currentUser = getCurrentUserDetails();
 
     for (Enrollment enrollment : enrollments) {
-      if (enrollment != null
-          && (orgUnitMode == ALL
-              || trackerOwnershipManager.hasAccess(
-                  currentUser, enrollment.getTrackedEntity(), enrollment.getProgram()))
-          && trackerAccessManager.canRead(currentUser, enrollment, orgUnitMode == ALL).isEmpty()) {
-        enrollmentList.add(getEnrollment(enrollment, fields, includeDeleted));
-      }
+      enrollmentList.add(getEnrollment(enrollment, fields, includeDeleted));
```

Comment:
Now we could just simplify this and use stream
`return enrollments.stream().map(e -> getEnrollment(enrollment, fields, includeDeleted)).toList()`